catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/SortedQueryMonitor.java

```diff
@@ -191,22 +199,22 @@ public void run() {
     }
 
     private void timeoutRemainingSources(Set<ProcessingDetails> processingDetails) {
-        for (Source expiredSource : futures.values()) {
+        for (QueryRequest expiredSource : futures.values()) {
             if (expiredSource != null) {
-                LOGGER.info("Search timed out for {}", expiredSource.getId());
-                processingDetails.add(new ProcessingDetailsImpl(expiredSource.getId(),
+                LOGGER.info("Search timed out for {}", getSourceIdFromRequest(expiredSource));
```

Comment:
nit: you could just get the sourceId once instead of calling getSourceIdFromRequest twice.